include/inviwo/core/datastructures/image/image.h

```diff
@@ -49,8 +49,7 @@ class IVW_CORE_API Image : public DataGroup<Image, ImageRepresentation>, public
     using DataBuffer = std::unique_ptr<std::vector<unsigned char>>;
 
     Image(size2_t dimensions = size2_t(8, 8), const DataFormatBase* format = DataVec4UInt8::get());
-    Image(std::shared_ptr<Layer> color, std::shared_ptr<Layer> depth,
-          std::shared_ptr<Layer> picking);
+    Image(std::vector<std::shared_ptr<Layer>> layers);
```

Comment:
This requires more explanation of what is expected and what happens in the different cases. E.g. that layers are created if not provided and that it requires that one layer is a color layer.

Why not also keep the previous explicit one?

Reply:
added some comments, the old was also added in this PR, and is a bit bad since it encodes the LayerType both in the parameter name, and in the layer it self.